source/03-components/accordion/accordion.es6.js

```diff
@@ -0,0 +1,170 @@
+import Drupal from 'drupal';
+import { slideDown, slideUp } from '../../06-utility/_slide.es6';
+
+Drupal.behaviors.accordion = {
+  attach(context) {
+    const ACCORDION_CLASS = 'js-accordion';
+    const ACCORDION_TOGGLE_CLASS = 'js-accordion-toggle';
+    const ACCORDION_SPEED = 250;
```

Comment:
These variable names should be camel case.

Reply:
@kmonahan This wasn’t flagged by ESLint or Codacy, which I’m surprised by. I guess I didn’t know our linting standards as well as I thought! :)

I’m fine with leaving it as-is.

Reply:
👍